en-US.json

```diff
@@ -700,50 +700,50 @@
 	"WAITING_FOR_USER_ACTION": "Please wait while processing",
 	"ERR_GETTING_OWNER_IDENTITIES": "Error getting owner idenities: ${0}",
-	"IDENTITY_OPTION_DATA": "identity: ${0}, privileges: ${1}, balance: ${2} ${3}",
-	"SELECT_IDENTITY": "Select identity",
+	"IDENTITY_OPTION_DATA": "account: ${0}, privileges: ${1}, balance: ${2} ${3}",
+	"SELECT_IDENTITY": "Select account",
 	"SELECT_IDENTITY_INFO": "Select identiy to connect with",
-	"FULL_LOGIN_INFO": "Select identity contract to create session",
-	"LOGIN_STANDARD_IDENTITY": "Select identity",
-	"ACCOUNT_SET_IDENTITY_PRIVILEGE_BTN": "Set identity privileges",
-	"ACCOUNT_SET_IDENTITY_PRIVILEGE_TITLE": "Set identity privileges",
+	"FULL_LOGIN_INFO": "Select account contract to create session",
+	"LOGIN_STANDARD_IDENTITY": "Select account",
+	"ACCOUNT_SET_IDENTITY_PRIVILEGE_BTN": "Set account privileges",
+	"ACCOUNT_SET_IDENTITY_PRIVILEGE_TITLE": "Set account privileges",
 	"ACCOUNT_SET_IDENTITY_PRIVILEGE_STEP": "Privileges data",
-	"SET_IDENTITY_PRIVILEGE_MAIN_INFO": "You can authorize current and other wallet addresses to interact with this identity contract. You can set different levels of access including removing it. To change identity privileges you need at least \"Transactions\" privilege level. Please be careful when using this action.",
+	"SET_IDENTITY_PRIVILEGE_MAIN_INFO": "You can authorize current and other wallet addresses to interact with this account contract. You can set different levels of access including removing it. To change account privileges you need at least \"Transactions\" privilege level. Please be careful when using this action.",
 	"ADDR_TO_SET_PRIV_LEVEL": "Wallet address to set privilege level",
 	"ADDR_TO_SET_PRIV_LEVEL_HELPER_TXT": "Checksumed ETH address is required",
 	"SELECT_PRIV_LEVEL": "Select privilegе level",
 	"SELECT_PRIV_LEVEL_HELPER_TXT": "Choose wisely",
 	"ACCOUNT_SET_IDENTITY_PRIVILEGE_SAVE_BTN": "Set privileges",
 	"ACCOUNT_ADVANCED_INFO_AND_ACTIONS": "Advanced",
 	"MANAGE_IDENTITY": "Manage idenity",
-	"ERR_IDENTITY_SET_ADDR_PRIV_NOTIFICATION": "Error setting identity privileges: ${0}",
-	"IDENTITY_SET_ADDR_PRIV_NOTIFICATION": "Identity privileges transactions sent!",
-	"INSUFFICIENT_BALANCE_FOR_FEES": "Insufficient balance for transaction fees. Identity avaialbe: ${0} ${1}, fees: ${2} ${3}",
+	"ERR_IDENTITY_SET_ADDR_PRIV_NOTIFICATION": "Error setting account privileges: ${0}",
+	"IDENTITY_SET_ADDR_PRIV_NOTIFICATION": "Account ID privileges transactions sent!",
+	"INSUFFICIENT_BALANCE_FOR_FEES": "Insufficient balance for transaction fees. Account balance avaialbe: ${0} ${1}, fees: ${2} ${3}",
 	"PRIV_LEVEL_INFO_AND_FEES": "You will pay ${0} ${1} from idenitity for this transaction",
 	"PROP_SETADDR": "Address to set privilege",
 	"PROP_PRIVLEVEL": "Privilege level",
-	"ERR_IDENTITY_WITHDRAW_NOTIFICATION": "Error on withdrow from identity: ${0}",
+	"ERR_IDENTITY_WITHDRAW_NOTIFICATION": "Error on withdrow from account: ${0}",
```

Comment:
Another typo from before, `withdrow` should be `withdraw`

Reply:
Nice catch again